samples/drivers/led_ws2812/boards/thingy52_nrf52832.conf

```diff
@@ -0,0 +1,5 @@
+CONFIG_SPI=n
+
+CONFIG_I2S=y
+CONFIG_WS2812_STRIP=y
+CONFIG_WS2812_STRIP_I2S=y
```

Comment:
I think the sample is getting a bit messier.  And it already was some :)

We have configuration and overlay files using different drivers. But it is impossible to tell which one from the filenames...

@mbolivar-nordic, probably there is something to do here ?

Reply:
should we add a section in the readme for each? at least one for each driver.

Reply:
I am not sure how we should deal with that. @mbolivar-nordic, what do you think ?

Reply:
added.